code/modules/mob/living/carbon/human/human.dm

```diff
@@ -949,10 +949,24 @@
 	return (ishuman(target) && !(target.mobility_flags & MOBILITY_STAND))
 
 /mob/living/carbon/human/proc/fireman_carry(mob/living/carbon/target)
+	var/carryspeed = 0 //if you have latex you are faster at grabbing
+	if(HAS_TRAIT(src, TRAIT_QUICKER_CARRY))
+		carryspeed = 35
+	else if(HAS_TRAIT(src, TRAIT_QUICK_CARRY))
+		carryspeed = 40
+	else 
+		carryspeed = 50
 	if(can_be_firemanned(target))
-		visible_message("<span class='notice'>[src] starts lifting [target] onto their back...</span>",
+		if(carryspeed == 35)
+			visible_message("<span class='notice'>[src] starts expertly lifting [target] onto their back.</span>",
+			"<span class='notice'>Using your gloves' nanochips, you quickly start to lift [target] onto your back.</span>")
+		else if(carryspeed == 40)
+			visible_message("<span class='notice'>[src] starts quickly lifting [target] onto their back..</span>",

```

Comment:
if you are copypasting a message you need to make it more malleable using variables.

IE have a variable that is by default `" "//mind the spaces` then if they have the trait change it to `" quickly "` or `" expertly "` then
```DM
visible_message("<span class='notice'>[src] starts[adverb_here_var]lifting [target] onto their back..</span>",
```

Reply:
done